range_finding.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
```

Comment:
We are not using this in the current scheme please remove

Reply:
File removed